lib/ipc/common.c

```diff
@@ -91,7 +91,7 @@ _heim_ipc_create_cred(uid_t uid, gid_t gid, pid_t pid, pid_t session, heim_icred
     return 0;
 }
 
-#ifndef HAVE_GCD
+#if !defined(HAVE_GCD) && defined(ENABLE_PTHREAD_SUPPORT)
```

Comment:
Ah, I missed that probably because of the stub for heim_isemaphore in heim-ipc.h. It compiled, but you're right, this is wrong.